release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2716,7 +2716,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.15.9
+[id="ocp-4-15-10"]
+=== RHSA-2024:1887 - {product-title} 4.15.10 bug fix and security update
+
+Issued: 2024-04-25
+
+{product-title} release 4.15.10, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1887[RHSA-2024:1887] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1892[RHSA-2024:1892] advisory.
```

Comment:
link:https://access.redhat.com/errata/RHSA-2024:1892[RHSA-2024:1892] not currently working, but I do realize it just may not be live yet....

Reply:
It's not live yet. Thanks for checking!